server/datastore/mysql/migrations/tables/20231002120317_AddDiscardToQueries.go

```diff
@@ -0,0 +1,18 @@
+package tables
+
+import (
+	"database/sql"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20231002120317, Down_20231002120317)
+}
+
+func Up_20231002120317(tx *sql.Tx) error {
+	_, err := tx.Exec(`ALTER TABLE queries ADD COLUMN discard_data TINYINT(1) NOT NULL DEFAULT FALSE;`)
```

Comment:
Double checking:
For existing queries we want them to start storing results?

Reply:
yes, I believe we settled on that in a call yesterday

Reply:
This seems to contradict the requirements in the epic: https://github.com/fleetdm/fleet/issues/7766

![Screenshot 2023-10-04 at 12 29 11](https://github.com/fleetdm/fleet/assets/2073526/d8829a49-d045-49a3-bed8-9362db047806)

/cc @rachaelshaw @zhumo

Reply:
Just double checking, was this resolved in slack?

Reply:
Yes. This was fixed. Existing queries will be migrated with `discard_data` set to `true`.

Reply:
yes, was resolved. Thanks for checking @zhumo!